admin.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**this function is to get subtitle column data from the about_me table in the database
+ *
+ * @param $db array data to be selected from
+ *
+ * @return array of subtitle column values
+ */
+function getAboutMeSubtitleFromDatabase($db) {
+    $query = $db->prepare("SELECT `id`,`subtitle` FROM `about_me` WHERE `deleted` = 0;");
+    $query->execute();
+    return $query->fetchAll();
+}
+/**this function is to return the values of each subtitle in about_me section into a drop-down form
```

Comment:
pop a line break inbetween functions (do for all of them)

Reply:
cool, all done